clients/js/cmds/sui.ts

```diff
@@ -59,79 +107,139 @@ exports.builder = function(y: typeof yargs) {
+          type: "string"
         })
         .option("governance-chain-id", {
           alias: "gci",
           describe: "Governance chain ID",
-          default: 1,
-          type: "number",
+          default: "3",
+          type: "string",
           required: false
         })
         .option("governance-contract", {
           alias: "gc",
           describe: "Governance contract",
           type: "string",
-          default: "0x0000000000000000000000000000000000000000000000000000000000000004",
+          default: "0000000000000000000000000000000000000000000000000000000000000004",
           required: false
         })
         .option("initial-guardian", {
           alias: "ig",
           required: true,
-          describe: "Initial guardian public keys)",
+          describe: "Initial guardian public keys",
           type: "string",
         })
     }, async (argv) => {
       const network = argv.network.toUpperCase();
       assertNetwork(network);
       const rpc = argv.rpc ?? NETWORKS[network]["sui"].rpc;
-      const deployer = argv.deployer;
+      const package_id = argv["package-id"]
       const chain_id = argv["chain-id"];
       const governance_chain_id = argv["governance-chain-id"];
       const governance_contract = argv["governance-contract"];
       const initial_guardian = argv["initial-guardian"];
+      const provider = new JsonRpcProvider(rpc);
+      const signer = loadSigner(network, rpc);
+      const owner = await signer.getAddress()
+      const objects = await provider.getObjectsOwnedByAddress(
+        owner
+      );
+      const deployer = findDeployerCapability(package_id, "state", objects)
+      if (typeof deployer == 'undefined'){
+        throw new Error('Wormhole core bridge cannot be initialized because deployer capability cannot be found. Is the package published?');
+      }
+
       console.log("network: ", network)
       console.log("rpc: ", rpc)
-      console.log("deployer: ", deployer)
+      console.log("package id: ", package_id)
+      console.log("deployer object id: ", deployer)
       console.log("chain-id: ", chain_id)
       console.log("governance-chain-id: ", governance_chain_id)
       console.log("governance-contract: ", governance_contract)
       console.log("initial-guardian: ", initial_guardian)
+
       await callEntryFunc(
         network,
         rpc,
-        "packageObjectId",
+        String(package_id),
         "state",
         "init_and_share_state",
         [],
         [
           deployer,
           chain_id,
           governance_chain_id,
-          governance_contract,
-          [initial_guardian]
+          [...Buffer.from(governance_contract, "hex")],
+          [[...Buffer.from(initial_guardian, "hex")]]
         ],
     )
-      //await publishPackage(network, rpc, `${dir}/tokenbridge`);
     })
-    .command("init-coin", "Publish coin contract", (yargs) => {
+    .command("init-tokenbridge", "Init token bridge contract", (yargs) => {
```

Comment:
let's change this to `init-token-bridge` to match aptos
https://github.com/wormhole-foundation/wormhole/blob/4c7df41984b6f4947773846050bdd68efc3a8cff/clients/js/cmds/aptos.ts#L64